permalinks/upgrade_k8s.md

```diff
@@ -8,6 +8,7 @@ Kops will drop support for Kubernetes versions as follows:
 | 1.18         | 1.8 and below                        |
 | 1.19         | 1.9 and 1.10                         |
 | 1.20         | 1.11 and 1.12                        |
+| 1.21         | 1.13 and 1.14                        |
```

Comment:
Maybe we should add another column "Deprecates support for Kubernetes version" and use "Removes" instead of "Drops" in first, to match the release notes wording.